src/main/integrations/gpu-context.ts

```diff
@@ -0,0 +1,76 @@
+import { defineIntegration, dropUndefinedKeys, Event } from '@sentry/core';
+import { app } from 'electron';
+
+interface Options {
+  infoLevel: 'basic' | 'complete';
+}
+
+interface GpuContext extends Record<string, unknown> {
+  name: string;
+  active?: boolean;
+  vendor_id?: string;
+  vendor_name?: string;
+  device_id?: string;
+  driver_version?: string;
+}
+
+interface GpuDevice {
+  active: boolean;
+  deviceId: number;
+  deviceString?: string;
+  driverVendor?: string;
+  driverVersion?: string;
```

Comment:
I think `driverVersion` maps to `version` on the GPU context.

Reply:
I thought that might be the case until I saw the example which has `metal`:
https://develop.sentry.dev/sdk/data-model/event-payloads/contexts/#example-gpu-context

Reply:
@bruno-garcia do have any insight for this `version` property in the GPU context?